web/src/pages/Mountain.scss

```diff
@@ -0,0 +1,138 @@
+        background-position: center center;
+        background-size: cover;
+
+        margin: 0 var(--base-space-2x-negative);
+        height: 25vh;
+
+        display: flex;
+        align-items: center;
+
+        .trail-name {
+            background-color: var(--background-color-transparent);
+            backdrop-filter: blur(30px) saturate(130%);
+            backface-visibility: hidden;
+
+            padding: var(--base-space) var(--base-space-2x);
+            margin: 0 var(--base-space-2x);
+            border-radius: var(--border-radius);
+            box-shadow: var(--interactive-surface-box-shadow);
+
+            display: flex;
+            align-items: center;
+
+            .trail-text {
+                color: var(--font-color-bright);
+                text-shadow: 0 2px 5px rgba(0, 0, 0, 0.5);
+            }
+
+            .header-icon {
+                width: 50px;
+                height: 50px;
+                margin-right: var(--base-space-2x);
+                background-color: var(--staze-green);
+                padding: var(--base-space);
+                border-radius: 50%;
+
+                path {
+                    fill: var(--font-color-bright);
+                }
+            }
+        }
+    }
+
+    .content-wrapper {
+        display: flex;
+        flex-direction: row;
+        margin: 0 var(--base-space-2x-negative);
+
+        .trails-list {
+            width: 40%;
+            min-width: 500px;
+            background: var(--surface-background-color);
+
+            h1 {
+                padding: var(--base-space-2x);
+            }
+
+            .filters {
+                display: flex;
+                padding: 0 var(--base-space-2x) var(--base-space-2x);
+
+                button {
+                    border-radius: var(--card-border-radius);
+
+                    + button {
+                        margin-left: var(--base-space);
+                    }
+                }
+            }
+
+            .trails {
+                list-style-type: none;
+
+                .trail {
+                    padding: var(--base-space) var(--base-space-2x);
+                    border-bottom: 1px solid var(--border-color);
+
+                    .trails-about {
+                        display: flex;
+
+                        .trail-distance {
+                            width: 15%;
+                            min-width: 100px;
+                        }
+
+                        .trail-duration {
+                            width: 15%;
+                            min-width: 100px;
+                        }
+
+                        .trail-height-difference {
+                            width: 15%;
+                            min-width: 100px;
+                        }
+
+                        .trail-start {
+                            width: 55%;
+                        }
+
+                        .trail-specs {
+                            color: grey;
```

Comment:
I think we need another CSS variable for this, add a new one in Theme.scss (maybe --font-color-secondary?)